src/PowerShellEditorServices/Services/DebugAdapter/DebugService.cs

```diff
@@ -603,6 +603,59 @@ public VariableScope[] GetVariableScopes(int stackFrameId)
             };
         }
 
+        internal void SetPathMappings(PathMapping[] pathMappings) => _pathMappings = pathMappings;
+
+        internal void UnsetPathMappings() => _pathMappings = null;
+
+        internal bool TryGetMappedLocalPath(string remotePath, out string localPath)
+        {
+            if (_pathMappings is not null)
```

Comment:
If _pathMappings is initialized to an empty dictionary then foreach over it in that scenario is a no-op and we don't have to do this check (and nested statement).